src/libs/OptionsListUtils.ts

```diff
@@ -1297,6 +1335,12 @@ function getValidOptions(
         return true;
     });
 
+    let workspaceChats: ReportUtils.OptionData[] = [];
+
+    if (shouldSeparateWorkspaceChat) {
+        workspaceChats = allReportOptions.filter((option) => option.isPolicyExpenseChat);
```

Comment:
@JKobrynski i am not sure if this completely covers it but we should only show the workspace chats that are your own. So the isOwnPolicyExpenseChat property is true.

As admin you will have workspace chats locally of many other members that you cannot submit to